File: Iniciante/URI 2234.cs
using System; 
 
class URI {
     
    static void Main(string[] args) { 
        string[] input = Console.ReadLine().Split(' ');
        double H = double.Parse(input[0]);
        double P = double.Parse(input[1]);
        double media = H / P;
        Console.WriteLine("{0:F2}", media);
    }
    
}

File: Iniciante/URI 2234.dart
import 'dart:io';

void main() { 
  List<String> input = stdin.readLineSync().split(" ");
  double H = double.parse(input[0]);
  double P = double.parse(input[1]);
  double media = H / P;
  print("${media.toStringAsFixed(2)}");
}